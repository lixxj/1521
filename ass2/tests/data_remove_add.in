a = malloc 1000
b = malloc 1000
free a
c = malloc 969
d = malloc 1
free c
e = malloc 960
f = malloc 1