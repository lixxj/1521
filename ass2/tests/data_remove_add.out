+00000 (F, 5000) 
[a] +00008 
+00000 (A, 1008) +01008 (F, 3992) 
[a] +00008 [b] +01016 
+00000 (A, 1008) +01008 (A, 1008) +02016 (F, 2984) 
[b] +01016 
+00000 (F, 1008) +01008 (A, 1008) +02016 (F, 2984) 
[b] +01016 [c] +00008 
+00000 (A, 1008) +01008 (A, 1008) +02016 (F, 2984) 
[b] +01016 [c] +00008 [d] +02024 
+00000 (A, 1008) +01008 (A, 1008) +02016 (A,   12) +02028 (F, 2972) 
[b] +01016 [d] +02024 
+00000 (F, 1008) +01008 (A, 1008) +02016 (A,   12) +02028 (F, 2972) 
[b] +01016 [d] +02024 [e] +00008 
+00000 (A,  968) +00968 (F,   40) +01008 (A, 1008) +02016 (A,   12) +02028 (F, 2972)
[b] +01016 [d] +02024 [e] +00008 [f] +00976 
+00000 (A,  968) +00968 (A,   40) +01008 (A, 1008) +02016 (A,   12) +02028 (F, 2972)
+00000 (A,  968) +00968 (A,   40) +01008 (A, 1008) +02016 (A,   12) +02028 (F, 2972)
